doc/man/8/ceph.rst

```diff
@@ -1189,12 +1189,15 @@ Usage::
 quorum
 ------
 
-Enter or exit quorum.
+Cause a random MON to enter or exit quorum.
```

Comment:
To be specific, the random MON is the monitor which the "ceph" cli is currently talking to. @smithfarm @jecluis shall we note this down or just put "random"?